src/Perl6/Actions.nqp

```diff
@@ -1427,11 +1427,31 @@ class Perl6::Actions is HLL::Actions does STDActions {
         my $doc := %*COMPILING<%?OPTIONS><doc>;
         if $doc {
             my $block := $*W.push_lexpad($/);
-
-            my $renderer := "Pod::To::$doc";
-
-            my $module := $*W.load_module($/, $renderer, {}, $block);
-
+            my $rx := /
+                       ^
+                       $<name> = ( [ \w | '::' ] + )
+                       [ ':' $<part> = ( [ 'ver' | 'auth' | 'api' ] )
+                        \< ~ \> $<val> = ( .*? ) ]*
+                       $
+                      /;
+            my $s := match($doc, $rx);
+            my %opts := nqp::hash();
+            my @parts := nqp::atkey($s,'part');
+            my @vals := nqp::atkey($s,'val');
+            my $e := nqp::elems(@parts);
+
+            my $i := 0;
+            while $i < $e { nqp::bindkey( %opts, ~ @parts[$i], ~ @vals[$i++] ) };
+
+            my $renderer := ~ nqp::atkey($s,'name');
+            my $module;
```

Comment:
You sure that what atkey remembers is something other than a string?

Reply:
Yes. atkey returned matches for the parts and vals. That caused failures, so the ~ was needed there. I added for name, just incase.